Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -17,6 +17,31 @@ public final class ProjectPamphletViewController: UIViewController {
 
   @IBOutlet weak private var navBarTopConstraint: NSLayoutConstraint!
 
+  private let backThisProjectContainerViewMargins = Styles.grid(3)
+  private let backThisProjectContainerView: UIView = {
+    return UIView(frame: .zero) |> \.translatesAutoresizingMaskIntoConstraints .~ false
```

Comment:
I wonder if we should sooner or later start using a convenience functions / initializers to create our views that we'll be building programmatically. One of the pesky things is to keep remembering not to forget enabling auto layout which is by default off. This means each of our instance will have to have `translatesAutoresizingMaskIntoConstraints` set to `false`.

How would you folks feel about introducing something like this, that would allow us constructing views in more consistent way and we would not have to think about forgetting enabling auto layout?

<img width="1072" alt="Screen Shot 2019-04-15 at 12 41 42 PM" src="https://user-images.githubusercontent.com/387596/56160426-0a020a00-5f7c-11e9-8c1d-523ba8206e66.png">

Cc: @justinswart @Scollaco @cdolm92 

Note: By all means this should probably not be part of this PR

Reply:
I like it 😎